.generator/templates/model_utils.mustache

```diff
@@ -110,6 +110,18 @@ def composed_model_input_classes(cls):
 class OpenApiModel(object):
     """The base class for all OpenAPIModels"""
 
+    _composed_schemas = {}
+
+    additional_properties_type = None
+
+    attribute_map = {}
```

Comment:
I am not a big fan of mutable defaults. Can we use `None` and modify usage?

Reply:
or do we even need it?

Reply:
Yeah it's not great to have mutable defaults, but we don't update those anywhere so I think it's ok? Otherwise that makes for a bunch of places to check for None.